policy/modules/system/getty.te

```diff
@@ -47,8 +47,11 @@ allow getty_t getty_log_t:file { append_file_perms create_file_perms setattr_fil
 logging_log_filetrans(getty_t, getty_log_t, file)
 
 allow getty_t getty_runtime_t:dir watch;
+allow getty_t getty_runtime_t:file watch;
 manage_files_pattern(getty_t, getty_runtime_t, getty_runtime_t)
 files_runtime_filetrans(getty_t, getty_runtime_t, file)
+allow getty_t init_runtime_t:dir read;
+allow getty_t init_runtime_t:sock_file write;
```

Comment:
Youre not allowed to reference init_runtime_t directly here. This looks like this might be "io.systemd.DynamicUsers", if so then auth_use_nsswitch(getty_t) should address it if and when https://github.com/SELinuxProject/refpolicy/pull/269 is merged